resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js

```diff
@@ -49,6 +49,17 @@ const GalleryBlockSignup = ({ campaignId, path }) => {
       variables: {
         campaignId,
       },
+      onCompleted: () => {
+        trackAnalyticsEvent('completed_signup', {
+          action: 'signup_completed',
+          category: EVENT_CATEGORIES.campaignAction,
+          label: campaignTitle,
+          context: {
+            url: path,
+            contextSource: 'scholarship_card',
+          },
```

Comment:
Perhaps worth adding the page context as in the current click event?
```suggestion
          context: {
            url: path,
            contextSource: 'scholarship_card',
            ...getPageContext(),
          },
```

(I wonder if we should start just including this in the canonical [track event helper](https://github.com/DoSomething/phoenix-next/blob/66e61dda9e3cc13b29fae6845decf2817c7c4564/resources/assets/helpers/analytics.js#L287) 🤔 )